mdp/nodes/sfa_nodes.py

```diff
@@ -369,6 +378,200 @@ def get_eta_values(self, t=1):
         return self._refcast(t / (2 * numx.pi) * numx.sqrt(self.d))
 
 
+class VartimeSFANode(SFANode):
+    """
+    Extract the slowly varying components from the input data.
```

Comment:
I also saw docs like
```
"""blah
blah blah
"""
```
in mdp, i.e. opposed to
```
"""
blah
blahblah
"""
Which one do we want? We should decide for one variant and apply it consistently.

Reply:
I think the answer is here: https://www.python.org/dev/peps/pep-0008/#documentation-strings